MovingBlocks/Terasology
# Improving/Adding Spanish Translations - GCI

### Contains

I have translated 60 words to Spanish in the menu_es.lang file. 

### How to test

Start the game, go to Player settings, change the language to Spanish and navigate the menu pages to see the words I've translated.

### Note:

I have not completely translated the file and there are significant chunks of menu text that are not in the menu_es.lang file to be translated.
